packages/editor/src/components/post-format/style.scss

```diff
@@ -2,8 +2,14 @@
 	margin: $grid-unit-05 0 0 0;
 }
 
-.editor-post-format__panel {
-	display: flex;
-	flex-direction: column;
-	align-items: stretch;
+.editor-post-format__dialog .editor-post-format__dialog-content {
+	// sidebar width - popover padding - form margin
+	min-width: $sidebar-width - $grid-unit-20 - $grid-unit-20;
+	margin: $grid-unit-10;
+}
+
+.editor-post-format__options {
+	.components-base-control__field > .components-v-stack {
+		gap: $grid-unit-15;
```

Comment:
The gap between radio options is way too small by default in RadioControl, I wonder if this style should be something to elevate to the components package. cc @mirka

Reply:
In progress at #61696